src/webgpu/util/copy_to_texture.ts

```diff
@@ -74,14 +102,18 @@ export class CopyToTextureUtils extends GPUTest {
     );
   }
 
+  // MAINTENANCE_TODO: type GPUPredefinedColorSpace only have 'srgb', remove the
+  // 'display-p3' type after it is included in GPUPredefinedColorSpace.
   getExpectedPixels(
     sourcePixels: Uint8ClampedArray,
     width: number,
     height: number,
     format: RegularTextureFormat,
     srcPremultiplied: boolean,
     dstPremultiplied: boolean,
-    isFlipY: boolean
+    isFlipY: boolean,
+    srcColorSpace: GPUPredefinedColorSpace | 'display-p3' = 'srgb',
+    dstColorSpace: GPUPredefinedColorSpace = 'srgb'
```

Comment:
I only meant to suggest changing dstColorSpace, not srcColorSpace, since srcColorSpace is not in principle tied to GPUPredefinedColorSpace (it is actually tied to the canvas enum `PredefinedColorSpace` but I'm guessing that's not defined in TypeScript yet).
```suggestion
    srcColorSpace: 'srgb' | 'display-p3' = 'srgb',
    dstColorSpace: GPUPredefinedColorSpace = 'srgb'
```